test/integration_test/utils/px_operator.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 const (
-	nextReleaseTag = "1.9.0-dev"
+	nextReleaseTag = "1.8.1-dev"
```

Comment:
Should we leave this to 1.9.0 if 1.8.1 is going to be a quick hot fix release?

Reply:
this doesn't matter, the pipeline will use the default jenkins env parameter, I can update this again once 1.8.1 is released.